splice/web/api/adgroup.py

```diff
@@ -0,0 +1,130 @@
+from flask import Blueprint
+from flask_restful import Api, Resource, marshal, fields, reqparse
+
```

Comment:
I usually only see one empty line when separating imports like this. Just what I'm used to. If this is the way this project does it, that's fine. Just making sure it wasn't an accident.